### DISCUSSION TITLE
Compatibility with Python3?

### DISCUSSION THREAD

**Discussion by KevinKongWisc** [hernanchavezthielemann/GRO2LAM#49] [Q&A] [answered]
Hi, is there any possibility that a python3 version will be released? Thank you!

**Comment by hernanchavezthielemann** [comment, marked as answer]
Hi @KevinKongWisc,

Thanks for your question.
Indeed the released beta-beta version works just fine in python3.
I just have to do some formality checks and change a little the setup to ask for tkinter package in python3, but I think that package comes by default now, so GRO2LAM should work if you have a healthy installation of python3.
You can install the beta-beta with the following command line in Linux:

`wget https://raw.githubusercontent.com/hernanchavezthielemann/utils/master/grotolam/G2L_iBeta && bash G2L_iBeta`

Best regards,
Hernan

**Comment by hernanchavezthielemann** [comment]
@KevinKongWisc  ?

**Reply by KevinKongWisc** [reply]
Thank you so much! The beta version works with python3!

**Reply by hernanchavezthielemann** [reply]
Great!
Thank you for letting me know.
Best,
Hernan.
